sdk/containerapps/Azure.ResourceManager.AppContainers/src/Custom/Models/ContainerAppVnetConfiguration.Serialization.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ClientModel.Primitives;
+using System.Runtime.CompilerServices;
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.ResourceManager.AppContainers.Models
+{
+    [CodeGenSerialization(nameof(InfrastructureSubnetId), DeserializationValueHook = nameof(DeserializeInfrastructureSubnetIdValue))]
+    public partial class ContainerAppVnetConfiguration : IUtf8JsonSerializable, IJsonModel<ContainerAppVnetConfiguration>
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static void DeserializeInfrastructureSubnetIdValue(JsonProperty property, ref ResourceIdentifier infrastructureSubnetId)
+        {
+            if (property.Value.ValueKind != JsonValueKind.Null && !string.IsNullOrEmpty(property.Value.GetString()))
+            {
+                infrastructureSubnetId = new ResourceIdentifier(property.Value.GetString());
+            }
```

Comment:
we could introduce a bit optimization:
```suggestion
            if (property.Value.ValueKind == JsonValueKind.Null)
            	return;
            var str = property.Value.GetString();
            if (!string.IsNullOrEmpty(str))
            {
                infrastructureSubnetId = new ResourceIdentifier(property.Value.GetString());
            }
```

Reply:
Or if there is a method `property.Value.TryGetString()` we could use it to avoid that we call `GetString` twice

Reply:
It seems that there is no method `property.Value.TryGetString()`.